### PR TITLE
Rename -jre flavor to openjdk flavor

### PR DESCRIPTION
**Pull request by dirkmueller**

### CODE REVIEW

File: src/bci_build/package/apache_tomcat.py [hunk 1]
```diff
@@ -66,13 +66,13 @@ def _get_sac_supported_until(
             and os_version.is_tumbleweed
         ),
         version="%%tomcat_version%%",
-        tag_version=f"{tomcat_major}-jre{jre_version}",
+        tag_version=f"{tomcat_major}-openjdk{jre_version}",
```
Comment (alexandrevicenzi) [inline src/bci_build/package/apache_tomcat.py:69]: nit: rename `jre_version` to `jdk_version` since we are talking about JDK and not JRE.

**Review by alexandrevicenzi — APPROVED** [review]

File: src/bci_build/package/apache_tomcat.py (hunk 1 above)
Comment (dirkmueller) [reply src/bci_build/package/apache_tomcat.py:69]: well, we still only install the JRE not the JDK (but we name the tag suffix jdk which I find a bit confusing as well)..

Comment (dcermak) [reply src/bci_build/package/apache_tomcat.py:69]: Why are you proposing to change the tag suffix then if it's now more confusing?

Comment (alexandrevicenzi) [reply src/bci_build/package/apache_tomcat.py:69]: the package list is `f"java-{jre_version}-openjdk", f"java-{jre_version}-openjdk-headless"`, is this JRE only? I thought it was JDK

Comment (dirkmueller) [reply src/bci_build/package/apache_tomcat.py:69]: no, the jdk is the `java-{jre_version}-openjdk-devel`

**Pull request merged**